packages/flutter/lib/src/rendering/stack.dart

```diff
@@ -49,6 +49,7 @@ class RelativeRect {
     );
   }
 
+  /// A relative rect that covers the entire rect.
```

Comment:
Changed to `container`, which is defined terminology for this class (according to the class-level docs).